rdb_writer.go

```diff
@@ -0,0 +1,35 @@
+package rdb
+
+type RDBWriter interface {
```

Comment:
what is the point of this interface? i only see a single implementation

Reply:
Initially, I planned to implement an in-memory writer for testing, but it isn't worth the effort and is meaningless.